.github/workflows/validate-pr.yaml

```diff
@@ -108,3 +121,74 @@ jobs:
         with:
           number: ${{ github.event.pull_request.number }}
           path: code-coverage-results.md
+
+  release:
+    name: 🐳 Build & publish Docker image
+    runs-on: ubuntu-latest
+    needs:
+      - lint
+      - type-check
+      - test
+    env:
+      image-name: gh-mpyl
+      image-version: pr-${{ github.event.pull_request.number }}
+    outputs:
+      image: ${{ steps.login-ecr-public.outputs.registry }}/vdb-public/${{ env.image-name }}:${{ env.image-version }}
+    steps:
+      - uses: actions/checkout@v4.2.1
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4.0.2
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ECR_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_ECR_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
```

Comment:
Why this one and something in the EU (like we do in other actions as well)?

Reply:
This is where we publish [all our public images](https://github.com/Vandebron/docker-images/blob/main/.github/workflows/on-main.yaml#L66-L71).

Reply:
Okay, I searched in the actions repo. My question is still why, but I guess you don't know since you didn't decide it